### PR TITLE
Add nearby command

### PR DESCRIPTION
**Pull request by khooroko**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -135,6 +136,10 @@ public Command parseCommand(String userInput) throws ParseException {
             logger.info("----------------[ACTUAL COMMAND][" + commandWord + "]");
             return new BorrowCommandParser().parse(arguments);
 
+        case NearbyCommand.COMMAND_WORD:
```
Comment (jaivigneshvenugopal) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:139]: Should we have an alias for nearby? Actually does not really matter.

**Review by jaivigneshvenugopal — APPROVED** [review]
Woah this is really nice!

File: src/main/java/seedu/address/logic/commands/NearbyCommand.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+import java.util.List;
+
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.events.ui.JumpToNearbyListRequestEvent;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.ReadOnlyPerson;
+
```
Comment (jelneo) [inline src/main/java/seedu/address/logic/commands/NearbyCommand.java:11]: Don't forget your author tag :)

File: src/main/java/seedu/address/ui/NearbyPersonListPanel.java [hunk 3]
```diff
@@ -69,6 +72,12 @@ private void scrollTo(int index) {
         });
     }
 
+    @Subscribe
+    private void handleJumpNearbyToListRequestEvent(JumpToNearbyListRequestEvent event) {
```
Comment (jelneo) [inline src/main/java/seedu/address/ui/NearbyPersonListPanel.java]: Perhaps the function could be `handleJumpToNearbyListRequestEvent` for clarity

File: src/test/java/seedu/address/ui/NearbyPersonListPanelTest.java [hunk 4]
```diff
@@ -55,4 +56,15 @@ public void display() {
         assertCardDisplaysPerson(expectedPerson, actualCard);
         assertEquals(Integer.toString(1) + ". ", actualCard.getId());
     }
+
+    @Test
+    public void handleJumpToNearbyListRequestEvent() {
```
Comment (jelneo) [inline src/test/java/seedu/address/ui/NearbyPersonListPanelTest.java:61]: same issue as above about the naming of function

**Review by jelneo — APPROVED** [review]
This is no small feat. Great job! Keep it up!

File: src/main/java/seedu/address/ui/NearbyPersonListPanel.java (hunk 3 above)
Comment (khooroko) [reply src/main/java/seedu/address/ui/NearbyPersonListPanel.java]: good catch

File: src/test/java/seedu/address/ui/NearbyPersonListPanelTest.java (hunk 4 above)
Comment (khooroko) [reply src/test/java/seedu/address/ui/NearbyPersonListPanelTest.java:61]: this one seems okay to me though

**Review by lawwman — APPROVED** [review]
Good job implementing the command!

**Pull request merged**